Chapters/1-5-Chapter5.md

```diff
@@ -0,0 +1,155 @@
+
+_The timekeeper. I remember now._
+
+I helped Fajro walk from the railyard to the easternmost funicular station. It was not terribly distant, but the sluggish pace was wearing thin on the patience of passersby trying to get around us.
+
+"I regret that I am slowing you down," he said, acknowledging his frustration.
+
+"No such talk. These people can afford to slow down a little bit."
+
+"You know, I used to keep pace with the trainferriers before I busted my knee."
+
+"Oh I bet you could run circles around them," I said, trying to lift his spirits a bit.
+
+We rode the funicular up the mountain and toward the Senate. It made several stops along the way, giving us places to stop and marvel at the beauty of the city from above. The public workers with their dusty canvas tunics, the scribes with their ink-blotted fingers, all funneling in and out of the car. Some were carrying their afternoon meal with them as if they had no time to eat, and it made me terribly hungry, despite my anxiety. Compartmentalizing everything I felt was no small feat, but there wasn't much of a choice. This had to be done. We had to prepare for what was to come.
+
+"Senate. Last stop," called the operator.
+
+I helped him off and he took a seat on the first bench he could find to rest his legs. It was a small public square directly in front of the senate building.
+
+"Ah this brings me back. You see that gate? I used to walk through it every day. I would always round the left corner as I entered from the west. And the steam fountain - Oh I loved to perch on the side of that thing and rake my fingers through the cool mist."
+
+"You sound very fond of this place."
+
+"You could say that. Almost all of my waking life was devoted to the people here. It really made an impression on me."
+
+"Do you ever miss it?"
+
+"Some parts. I felt proud of myself, and all the work that we put in to make big advancements. You know, I played a big part in the unification pact in Holrag."
+
+"You?"
+
+"I brokered a treaty for the tribes of Holragi to work together. The pact helped us become the manufacturing giant we are today, and deployed hundreds of public workers to design and survey the space rail."
+
+"You must be proud," I said, not really caring.
+
+"It would be a sin not to be proud. Alas, the scientists who headed the program won't allow me that privelege. They wanted to expand the observer program; to send explorers with telescopes into space to see what we cannot from the surface. By the end, we had only built the rail to half the length we needed, so it was only long enough to send lightweight things into orbit. Even the smallest craft they wanted to send would be too heavy."
+
+"Ideas never make it to reality unscathed," I said. "I learned that one early."
```

Comment:
So given my profession, I agree. But I am not sure what in Liffa's life would give hae this lesson, especially at this point in the story and what we have learned so far about this character.

Reply:
Ah I was thinking hae chance to be an observer that was stopped by hae parents' untimely death. in my re-write I'm making this a bigger deal.